components/app/dao-project-overview/DaoProjectsHeader.vue

```diff
@@ -1,72 +1,85 @@
-                <p class="text-primary small-text text-primary truncate">{{ card.title | truncate(18) }}</p>
-                <p class="small-text">{{ card.daoInfo }}</p>
-              </div>  
+            <div class="mr-3">
+              <img
+                class="inline-block mt-3 h-6 w-6 rounded-full ring-2 ring-white"
+                :src="card.imageURL"
+                :alt="card.title"
+              />
             </div>
+            <div>
+              <p class="text-primary small-text truncate">
+                {{ card.title | truncate(18) }}
+              </p>
+              <p class="small-text">{{ card.daoInfo }}</p>
+            </div>
+          </div>
+        </div>
       </div>
-     </div>
     </div>
-    <hr class="text-primary my-16">
+    <hr class="text-primary my-16" />
   </LandingSectionContainer>
 </template>
 
 <script>
-  import LandingSectionContainer from '../landing/LandingSectionContainer'
-  export default {
-    name: 'DaoProjectsHeader',
+import LandingSectionContainer from '../landing/LandingSectionContainer'
+export default {
+  name: 'DaoProjectsHeader',
 
-    components: {
-      LandingSectionContainer,
-    },
-    data() {
-      return {
-        daoInfoCards: [
-          {
-            title: 'Funding Round',
-            daoInfo: 'Round 6',
-            imageURL: require('@/assets/images/icons/fund-round.svg'),
-          },
-          {
-            title: 'Amount proposals',
-            daoInfo: '24 projects',
-            imageURL: require('@/assets/images/icons/amount.svg'),
-          },
-          {
-            title: 'Countdown',
-            daoInfo: 'ends in 4 days',
-            imageURL: require('@/assets/images/icons/countdown.svg'),
-          },
-          {
-            title: 'Total Amount',
-            daoInfo: '1800000 OCEAN',
-            imageURL: require('@/assets/images/icons/transaction.svg'),
-          },
-          {
-            title: 'Total Votes',
-            daoInfo: '30032992 OCEAN',
-            imageURL: require('@/assets/images/icons/vote.svg'),
-          },
-        ],
-      }
+  components: {
+    LandingSectionContainer,
+  },
+  props: {
+    daoData: {
+      type: Object,
+      required: true,
     },
-  }
+  },
+  data() {
+    return {
+      daoInfoCards: [
+        {
+          title: 'Funding Round',
+          daoInfo: `${this.daoData.metrics.fundingRound} OCEAN`,
+          imageURL: require('@/assets/images/icons/fund-round.svg'),
+        },
+        {
+          title: 'Amount proposals',
+          daoInfo: `${this.daoData.metrics.totalDaoProposals} proposals`,
+          imageURL: require('@/assets/images/icons/amount.svg'),
+        },
+        {
+          title: 'Countdown',
+          daoInfo: this.daoData.metrics.endDate,
+          imageURL: require('@/assets/images/icons/countdown.svg'),
+        },
+        {
+          title: 'Total Amount',
+          daoInfo: `${this.daoData.metrics.totalRequestedFunding} OCEAN`,
```

Comment:
chore(blocking): you added 'general.ocean' as new translation. Why are you not using it here? Consistency is key.